src/lib/components/AnnouncementBanner/AnnouncementBanner.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import "./AnnouncementBanner.scss";
+  import Link from "$lib/components/Link";
+  import Icon from "$lib/components/Icon";
+  import { url as lightbulbIcon } from "$icons/lightbulb";
+  import { url as arrowIcon } from "$icons/arrow_forward";
+  import classNames from "$lib/util/classNames";
+
+  export let headerText = "Here's a site-wide announcement!";
+  export let linkText = "Read more here";
+  export let href = "/";
+  let classes = classNames("usa-banner", `ldaf-banner--announcement`);
+</script>
+
+<section id="top" class={classes} aria-label="Official website of the Louisiana state government">
```

Comment:
Probably need to change this `aria-label` so that it's something generic, and maybe make this something we can optionally set as a prop.